### PR TITLE
readme.md

### PR DESCRIPTION
**Pull request by yashgautam9274**
#22 
added code of conduct file and link of that in readme.md 
contiburation guidelines and  link of that in readme.md 
and many new thing in readme.md

### CODE REVIEW

**Comment by yashgautam9274** [comment]
is there any mistake in my code

**Comment by garimasingh128** [comment]
I am still to review it. Please message on gitter if u have any queries.

**Comment by yashgautam9274** [comment]
@garimasingh128  all the changes done now you can merge my pull request

File: README.md [hunk 1]
```diff
@@ -49,14 +88,94 @@ You are also supposed to implement a Database in the Backend which should store
...
+-  JavaScript: 92.9%
+-  CSS: 3.9%
+-  HTML: 3.2%
+
+
+ ## 💻 Development guidelines
+
+1.  Put all the code in one of the existing files in  `/src`.
+    
+3.  Push all the code to your own branch. Once you are sure it is working, merge it with the `dev`  branch. Let's maintain only the stable and released versions on the  `master`  branch.
```
Comment (garimasingh128) [inline README.md:132]: change it to develop branch

File: README.md [hunk 2]
```diff
@@ -129,5 +248,50 @@ This section has moved here: https://facebook.github.io/create-react-app/docs/de
...
+> Made By Garima Singh with ❤️
+
+> Garima Singh &copy; 2020
+<br><br>
+[![ForTheBadge built-with-love](http://ForTheBadge.com/images/badges/built-with-love.svg)](https://github.com/garimasingh128/)
+[![ForTheBadge built-by-developers](http://ForTheBadge.com/images/badges/built-by-developers.svg)](https://github.com/garimasingh128/)
+
+## ❤️ Thanks to our awesome contributors.
+
+## Contributors ✨
```
Comment (garimasingh128) [inline README.md:287]: remove this part

File: README.md [hunk 3]
```diff
@@ -49,14 +88,94 @@ You are also supposed to implement a Database in the Backend which should store
...
+ 
+ ➡️   [Learning Resources](#-Learning-Resources)
+ 
+ ➡️   [Code Of Conduct](#--code-of-conduct)
+
+ ➡️   [License](#license)
+ 
+ ➡️   [Contributing](#contributors)
+
+## ❤️ Inspiration
```
Comment (garimasingh128) [inline README.md:116]: Add some more functions od todo apps. You can easily find them on the web.

**Comment by garimasingh128** [comment]
- change base branch to develop as its already mentioned in readme
- content looks fine acc to me. I would request you to check it once again for any minor errors
- fill up the PR template correctly with the changes done and the issues solved
- I have left some comments here: https://github.com/garimasingh128/ToDoista/pull/22/files
please resolve them

**Comment by yashgautam9274** [comment]
@garimasingh128  I send PR . and All changes completed

**Pull request closed without merging**